app/helpers/application_helper/toolbar_chooser.rb

```diff
@@ -446,7 +446,7 @@ def center_toolbar_filename_classic
               ems_container ems_middleware container_project container_route container_replicator container_image
               ems_network security_group floating_ip cloud_subnet network_router
               container_image_registry ems_infra flavor host container_build
-              ontap_file_share ontap_logical_disk container_topology middleware_topology
+              ontap_file_share ontap_logical_disk container_topology middleware_topology middleware_server
```

Comment:
@abonas I commented with @mtho11 that perhaps in some preliminar steps we are working in parallel with several efforts that might dependent. For example, touching the same files, so, I first idea would be for early stages to consolidate the "hawkular" efforts in a branch. In this way it could be more easy to merge/rebase these efforts. I am thinking that perhaps @mtho11 needs something that is currently under a different PR, so it seems that and order of PR would be needed. I think that complexity can be handled better in a different branch, instead that in master. Of course this approach could be limited on time, until we are in a development phase where tasks can be handled more independent way.
Also, it is just an idea, perhaps the style of work in MiQ is different and there are aspects we don't see.